Resources/Textures/Objects/Consumable/Food/Baked/pie.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c6e3401f2e7e1e55c57060cdf956a98ef1fefc24, mime tart slice and banana cream pie slice from rosysyntax under under CC BY-SA 4.0",
+    "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c6e3401f2e7e1e55c57060cdf956a98ef1fefc24, the pumpkin and pumpkin slice are made by ss14nekow (Mute Maid) and are under CC BY-SA 3.0, mime tart slice and banana cream pie slice from rosysyntax and are under CC BY-SA 3.0",
```

```suggestion
    "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c6e3401f2e7e1e55c57060cdf956a98ef1fefc24, the pumpkin and pumpkin slice are made by ss14nekow (Mute Maid), mime tart slice and banana cream pie slice from rosysyntax",
```
All the same licence now mentioned in the line above

---

good point 😆